script.js

```diff
@@ -5,19 +5,55 @@ var game = {
   players: [],
   hands: [],
   buildDeck: function(){
-
+    for (var v = 0; v < values.length; v++){
+      for (var s = 0; s < suits.length; s++){
+      game.deck.push({value: values[v], suit: suits[s], assignedValue: v});
+      }
+    }
+    return game.deck
   },
   shuffleDeck: function(){
-
+    for (var v = game.deck.length - 1; v > 0; v -= 1){
+      s = Math.floor(Math.random() * game.deck.length)
+      game.deck.push(game.deck[s]);
+      game.deck.splice(s, 1);
+    }
+    return game.deck
   },
   getMorePlayers: function(){
-
+    var newPlayer = prompt("Would you like to add an additional player?")
+    switch(newPlayer){
```

Comment:
Usually you only want a `switch` if you're comparing multiple things. But here you're just comparing one thing: whether it's `Yes`.